libs/index.md

```diff
@@ -70,6 +70,10 @@ Advanced usage (for badge makers):
       ['latest stable release', '/github/release/babel/babel/stable'],
       ['latest tag', '/github/tag/micromatch/micromatch'],
       ['watchers', '/github/watchers/micromatch/micromatch'],
+      ['status', '/github/status/micromatch/micromatch'],
+      ['status', '/github/status/tunnckoCore/badgen-service/github-status'],
```

Comment:
This is a "status branch" example, duplicated with "micromatch/micromatch/dev", and I guess you gonna delete it after PR merged? @tunnckoCore

Reply:
Was going to mention after i PR-ed, that i wont deleting the repo and branch.

Reply:
Not sure what the status would be if I merged this one 😂